CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [veneur-prometheus](https://github.com/stripe/veneur/tree/master/cmd/veneur-prometheus) now has a `-p` option for specifying a prefix for all metrics. Thanks [gphat](https://github.com/gphat)!
 * New metrics `ssf.spans.tags_per_span` and `ssf.packet_size` track the distribution of tags per span and total packet sizes, respectively.
 * Our super spiffy logo, designed by [mercedes](https://github.com/mercedes-stripe), is now included at the top of the README!
+* Refactor internals to use a new intermediary metric struct to facilitate new plugins. Thanks [gphat](https://github.com/gphat)!
```

Comment:
YOU CAN'T JUST THANK YOURSELF, CORY.

Reply:
https://media2.giphy.com/media/yoJC2El7xJkYCadlWE/giphy.gif